### PR TITLE
Add verification instructions. Closes #1300

### LINKED ISSUE #1300: Prompt user the other way of verification when verifying in one side
### Describe the problem

I hosted cinny in my webserver and it's surprisingly good! The only thing I confused about is that unlike in Element where you can start emoji verification on both side, you can only start it on the verified side. On the unverified side you can only fill in backup code.

### Describe the solution you'd like

At least prompt user to go to the other side and verify if they prefer.
It's a problem as simple as two lines of text! I can send a PR later.

### Alternatives considered

Allow to start emoji verification on either side. I'm not very clear about the spec and don't know whether it will work.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ZeroAurora**
### Description

Add verification instructions to the session list.

Screenshot:

图片

Fixes #1300

Also noticing that there are style and lint warnings in other files, I would like to remind maintainers about them :)

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by ZeroAurora** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/app/organisms/settings/DeviceManage.jsx [hunk 1]
```diff
@@ -217,6 +217,26 @@ function DeviceManage() {
     <div className="device-manage">
       <div>
         <MenuHeader>Unverified sessions</MenuHeader>
+        {!isMeVerified && (
+          <div style={{ padding: 'var(--sp-extra-tight) var(--sp-normal)' }}>
+            <InfoCard
+              rounded
+              variant="primary"
+              iconSrc={InfoIC}
+              title="Verify this session either with your Security Key here, or by logging into any of your verified sessions on other devices to verify with emoji."
```
Comment (kfiven) [inline src/app/organisms/settings/DeviceManage.jsx]: Can we make it:

Verify this session either with your Security Key/Phrase here or by initiating emoji verification from a verified session.

File: src/app/organisms/settings/DeviceManage.jsx [hunk 2]
```diff
@@ -217,6 +217,26 @@ function DeviceManage() {
...
+            />
+          </div>
+        )}
+        {isMeVerified && unverified.length > 0 && (
+          <div style={{ padding: 'var(--sp-extra-tight) var(--sp-normal)' }}>
+            <InfoCard
+              rounded
+              variant="surface"
+              iconSrc={InfoIC}
+              title="Verify other sessions by comparing emojis or remove unfamiliar sessions."
```
Comment (kfiven) [inline src/app/organisms/settings/DeviceManage.jsx]: This one:

Verify other sessions by emoji verification or remove unfamiliar ones.

**Comment by ZeroAurora** [comment]
@kfiven resolved

**Review by kfiven — APPROVED** [review]
Looks good, thanks for the PR 🎊

**Pull request merged**